charts/dremio/templates/dremio-coordinator.yaml

```diff
@@ -50,4 +50,7 @@ spec:
       initContainers:
       - name: wait-for-zk
         image: busybox
-        command:  ["sh", "-c", "until nc -z dremio-client 9047 > /dev/null; do echo waiting for dremio master; sleep 2; done;"]
+        env:
+        - name: COORDINATOR_WEB_PORT
+          value: "{{ .Values.coordinator.web.port | default 9047 }}"
+        command:  ["sh", "-c", "until nc -z dremio-client $(COORDINATOR_WEB_PORT) > /dev/null; do echo waiting for dremio master; sleep 2; done;"]
```

Comment:
You should be able to refer to the value {{ .Values.coordinator.web.port}} directly here without the need of an additional environment variable.